src/SideMenu.tsx

```diff
@@ -38,7 +44,14 @@ export const SideMenu = memo(
                 ref={ref}
                 style={style}
                 aria-labelledby="fr-sidemenu-title"
-                className={cx(fr.cx("fr-sidemenu"), className)}
+                className={cx(
+                    fr.cx("fr-sidemenu", {
+                        "fr-sidemenu--sticky": sticky === true,
```

Comment:
```suggestion
                        "fr-sidemenu--sticky": sticky,
```

Reply:
If we want a sticky aligned right menu, we have to mix both the two classes `fr-sidemenu--right` and `fr-sidemenu--sticky-full-height` ?

Reply:
I confirm that we don't need both classes at the same time to get the sticky menu working.
So stop me if I'm wrong but if we don't test the `sticky === true` we might have `fr-sidemenu--sticky` and `fr-sidemenu--sticky-full-height` present at the same time.